src/util/XDRCereal.cpp

```diff
@@ -0,0 +1,39 @@
+
```

Comment:
Missing license notice. I just realized the same applies for XDRCereal.h, so please fix it there too. Remember to update the year to 2020.

Reply:
Added!